### PR TITLE
[AF-2626] - Create Dashbuilder Runtime Image

### PR DESCRIPTION
**Pull request by spolti**
Signed-off-by: spolti <fspolti@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [x] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Attached commits represent units of work and are properly formatted
- [x] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [x] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Comment by spolti** [comment]
@jakubschwan @msmagnanijr @desmax74 @Kaitou786 this piece of the work is done, please review so we can merge it.

File: dashbuilder/modules/dashbuilder/module.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    value: "rhpam-7.10.0-dashbuilder-runtime.zip"
+ports:
+  - value: 8080
+artifacts:
+  - name: "ADD_ONS_DISTRIBUTION_ZIP"
+    target: "add_ons_distribution.zip"
+    # rhpam-7.10.0.redhat-20201101-add-ons.zip
+    md5: "7e285242ded7b9c7f1b508830b1a10ae"
+run:
+  user: 185
```
Comment (Kaitou786) [inline dashbuilder/modules/dashbuilder/module.yaml:32]: is this the UID which is needed by Dashbuilder to run?

Comment (spolti) [reply dashbuilder/modules/dashbuilder/module.yaml:32]: it is a pattern we've created in the past, this UDI refers to the "jboss" user.

**Review by Kaitou786 — APPROVED** [review]

**Comment by spolti** [comment]
@Kaitou786 just updated the artifacts with the latest build :D

**Review by Kaitou786 — APPROVED** [review]
Ah the circleCI check was missing, thanks :)

**Review by msmagnanijr — APPROVED** [review]

**Review by msmagnanijr — APPROVED** [review]

**Pull request merged**